src/pages/EnablePayments/PersonalInfo/substeps/AddressStep.tsx

```diff
@@ -43,7 +43,7 @@ function AddressStep({onNext, onMove, isEditing}: SubStepProps) {
             onMove={onMove}
             formID={ONYXKEYS.FORMS.WALLET_ADDITIONAL_DETAILS}
             formTitle={translate('personalInfoStep.whatsYourAddress')}
-            formPOBoxDisclaimer={translate('personalInfoStep.noPOBoxesPlease')}
+            formPOBoxDisclaimer={translate('common.noPo')}
```

Comment:
(Needs to be capitalized) 

```suggestion
            formPOBoxDisclaimer={translate('common.noPO')}
```